src/components/experiment-tracking/accordion/accordion.js

```diff
@@ -65,6 +122,21 @@ const Accordion = ({
             'accordion__title--medium': size === 'medium',
             'accordion__title--large': size === 'large',
           })}
+          onClick={() => {
+            if (size === 'medium') {
```

Comment:
should we introduce a new prop for sub-head, or a different name rather just relying on the size of it?

Reply:
Yeah, medium is confusing. I think we should have meaninful name like dataset